tests/energy_ratio_test.py

```diff
@@ -272,12 +350,76 @@ def test_alternative_weighting(self):
         # And energy ratio = 1.9 / 1
         self.assertAlmostEqual(er_out.df_result['wake_steering'].iloc[0], 1.9  , places=4)  
 
+    def test_weight_by_external_frequency_with_extra_df_freq_bin(self):
+
+        # Test that bins in df_freq which are not in data are ignored
+
+        df_base = pd.DataFrame({'wd': [270, 270., 270.,270.,],
+                           'ws': [7., 8., 8.,8.],
+                           'pow_000': [1., 1., 1., 1.],
+                           'pow_001': [1., 1., 1., 1.],
+        })
+
+        df_wake_steering  = pd.DataFrame({'wd': [270, 270., 270.,270.,],
+                           'ws': [7., 7., 8.,8.],
+                           'pow_000': [1., 1., 1., 1.],
+                           'pow_001': [2., 2., 1., 1.],
+        })
+
+        er_in = EnergyRatioInput([df_base, df_wake_steering],['baseline', 'wake_steering'], num_blocks=1)
+
+        # In the final test, specify a bin frequency where 7 m/s is 90% and 8 m/s is 10%
```

Comment:
Update the comment to "specify uniform bin frequencies"

Reply:
good catch